src/main/java/org/jenkinsci/extension_indexer/ExtensionPointListGenerator.java

```diff
@@ -304,7 +304,7 @@ private MavenRepositoryImpl createRepository() throws Exception {
      * Walks over the plugins, record {@link #modules} and call {@link #discover(Module)}.
      */
     private void processPlugins(MavenRepository r) throws Exception {
-        ExecutorService svc = Executors.newFixedThreadPool(4);
+        ExecutorService svc = Executors.newFixedThreadPool(1);
```

Comment:
Does this drastically increase the runtime?

Reply:
Appears to have been fixed in https://github.com/jenkins-infra/backend-extension-indexer/pull/35 by ripping out the affected code.

Testing locally with a patch using increased threads